.github/workflows/azure-ci.yml

```diff
@@ -4,11 +4,10 @@ on:
   # TODO: Remove after testing
   workflow_dispatch:
     inputs:
```

Comment:
@orouz, I believe the workflow_dispatch was initially used to test the Azure-CI workflow, and now that the workflow is functioning correctly, can we remove this trigger as well?